README.md

```diff
@@ -31,84 +42,58 @@ userPassword: readonly
 EOF
 ```
 
-Afterwards reload kidmd by running `systemctl reload kopano-kidmd`.
+And then passed as an additional parameter when starting `idmd` by passing `--ldif-config /etc/libregraph/idm/ldif/config.ldif`. The `config.ldif` is for service users only and the data in there is used for bind requests only, but never returned for search requests.
 
-The `config.ldif` is for service users only and the data in there is used for bind requests only, but never returned for search requests.
+#### Add users to the ldap service
 
-#### Main configuration
+`idmd` serves all ldif files from the folder specified by `--ldif-main` (loaded in lexical order and parsed as templates). Whenever any of the ldif files are changed, added or removed, make sure to restart `idmd`.
 
-The main directory data comes from the `main.d` folder. All `.ldif` files in there are loaded in lexical order and parsed as templates to create the data provided by the kidmd LDAP endpoint.
-
-Whenever any of the ldif files are changed, added or removed, make sure to reload kidmd with `systemctl reload kopano-kidmd`.
-
-Kidmd listens on `127.0.0.1:10389` by default and does not ship with any default users. Example configuration is installed (commented) and also available in the `/usr/share/doc/kopano-kidmd/examples` directory.
+`idmd` listens on `127.0.0.1:10389` by default and does not ship with any default users. Example configuration can be found in the [scripts directory](https://github.com/libregraph/idm/tree/master/scripts) of this repository.
 
 ##### Add new users using the `gen newusers` command
 
-Kidmd provides a way to create ldif data for new users using batch mode similar to the unix `newusers` command using the following standard password file format:
+IDM provides a way to create ldif data for new users using batch mode similar to the unix `newusers` command using the following standard password file format:
 
 ```bash
-uid:userPassword:uidNumber:gidNumber:cn,[mail][,kopanoAliases...]:ignored:ignored
+uid:userPassword:uidNumber:gidNumber:cn,[mail][,mailAlternateAddress...]:ignored:ignored
 ```
 
 For example, like this:
 
 ```bash
-cat << EOF | kopano-kidmd gen newusers - --min-password-strength=4 > /etc/kopano/kidm/ldif/main.d/50-users.ldif
-jonas:passwordOfJonas123:::Jonas Brekke,jonas@kopano.local::
+cat << EOF | ./idmd gen newuszxsedders - --min-password-strength=4 > /etc/libregraph/idm/ldif/main.d/50-users.ldif
```

Comment:
also here, make it generate a local `./users.ldif` or similar.

Reply:
yes, that is a good idea. I changed it to ./ldif/50-users.ldif to let the idea remain that it reads multiple files when specifying a folder